servers/cromwell/README.md

```diff
@@ -4,168 +4,181 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     ```
     export CROMWELL_URL=http://192.168.0.106:8000/api/workflows/v1
     ```
-
-1. (Optional) Fine-tune the parameters of Gunicorn server.
+- (Optional) Fine-tune the parameters of Gunicorn server.
     - By default, the shim layer uses **5** [Gunicorn workers](http://docs.gunicorn.org/en/stable/settings.html#worker-class). You can override the default number of workers by:
 
         ```
         export GUNICORN_CMD_ARGS="--workers=$NUMBER_OF_WORKERS"
-        ``` 
+        ```
         You can even use dynamic number of workers(based on the number of CPU cores), which is recommended, by:
         ```
         export GUNICORN_CMD_ARGS="--workers=$((2 * $(getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 2) + 1))"
         ```
 
     - By default, the shim layer uses **sync** [Gunicorn worker type](http://docs.gunicorn.org/en/stable/design.html#sync-workers). Since Job Manager also comes with `gevent` workers, you can override the default worker type by:
-    
+
         ```
         export GUNICORN_CMD_ARGS="--worker-class gevent"
         ```
-       
+
     - For convenience, you can consolidate the parameters in one command:
-    
+
         ```
-        export GUNICORN_CMD_ARGS="--workers=$((2 * $(getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 2) + 1)) --worker-class gevent"        
+        export GUNICORN_CMD_ARGS="--workers=$((2 * $(getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 2) + 1)) --worker-class gevent"
         ```
         before you run the shim container.
-    
-1. Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
-    ```
+
+- (Optional) Set up basic HTTP authentication
+
+  - Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
+    ```json
     {
       "cromwell_user" : "USERNAME",
       "cromwell_password" : "PASSWORD"
     }
     ```
 
-1. (Optional) If you want to change either the predefined view or the default behavior of UI to some extent, e.g. display more columns in the job list view such as labels of the jobs, or make more columns to be popped up by the query builder, you can add a `capabilities_config.json` file to `job-manager/servers/cromwell/jobs` to override the pre-defined configurations. The `capabilities_config.json` should **strictly** follow the following structure:
-    ```
+- (Optional for local Cromwell) Configure fields to display
+  - **Note**: CaaS-specific details follow in the next section. 
```

Comment:
I found this confusing; why refer to one section within a different section, when that first section is clearly labeled as "Required for CaaS"?